applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -174,9 +174,9 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def("GetInterfaceResidualSize", &TrilinosPartitionedFSIUtilitiesArray3DType::GetInterfaceResidualSize)
         .def("SetUpInterfaceVector", [](TrilinosPartitionedFSIUtilitiesArray3DType& self, ModelPart& rModelPart){
             return AuxiliaryVectorWrapper(self.SetUpInterfaceVector(rModelPart));})
-        .def("UpdateInterfaceValues", &AuxiliarUpdateInterfaceValues<array_1d<double,3>,3>)
```

Comment:
this changes the API ...

Reply:
AuxiliarUpdateInterfaceValues It is defined in the same file, cannot be accessed from other side, and is exported to python with a different name, doesn't break the API

Reply:
See the whole file, applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp, the definition is not accessible outside this file and exported with different name